src/client.rs

```diff
@@ -272,7 +272,14 @@ impl Client {
     pub async fn get_network_info(&self) -> Result<NetworkInfo> {
         let not_synced = self.network_info.read().map_or(true, |info| info.network_id.is_none());
 
-        if not_synced {
+        #[cfg(not(feature = "wasm"))]
+        let wasm = false;
+        #[cfg(feature = "wasm")]
+        let wasm = true;
```

Comment:
Would this be better?

```suggestion
        let wasm = if cfg!(feature = "wasm") { true } else { false };
```